Octokit.Reactive/Clients/ObservableUserGpgKeysClient.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+using System.Reactive.Threading.Tasks;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Octokit.Reactive.Internal;
+
+namespace Octokit.Reactive
+{
+    /// <summary>
+    /// A client for GitHub's UserUser GPG Keys API.
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="https://developer.github.com/v3/users/gpg_keys/">User GPG Keys documentation</a> for more information.
+    /// </remarks>
+    [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Gpg")]
+    public class ObservableUserGpgKeysClient : IObservableUserGpgKeysClient
+    {
+        readonly IUserGpgKeysClient _client;
+
+        public ObservableUserGpgKeysClient(IGitHubClient client)
+        {
+            Ensure.ArgumentNotNull(client, "client");
+
+            _client = client.User.GpgKeys;
+        }
+
+        /// <summary>
+        /// Gets all GPG keys for the authenticated user.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/users/gpg_keys/#list-your-gpg-keys">API documentation</a> for more information.
+        /// </remarks>
+        /// <returns>A <see cref="IReadOnlyList{GpgKey}"/> of <see cref="GpgKey"/>s for the current user.</returns>
+        public IObservable<GpgKey> GetAllForCurrent()
+        {
+            return GetAllForCurrent(ApiOptions.None);
+        }
+
+        /// <summary>
+        /// Gets all GPG keys for the authenticated user.
+        /// </summary>
+        /// <param name="options">Options for changing the API response</param>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/users/gpg_keys/#list-your-gpg-keys">API documentation</a> for more information.
+        /// </remarks>
+        /// <returns>A <see cref="IReadOnlyList{GpgKey}"/> of <see cref="GpgKey"/>s for the current user.</returns>
+        public IObservable<GpgKey> GetAllForCurrent(ApiOptions options)
```

Comment:
Nice pickup catering for the new API pagination options that were only recently introduced to octokit.net :+1: